sbom-generator/sbom_utils.py

```diff
@@ -0,0 +1,55 @@
+import hashlib
+from datetime import datetime
+
+def hash_sha1(file_path):
+    BLOCKSIZE = 65536
+    hasher = hashlib.sha1()
+    with open(file_path, 'rb') as afile:
+        buf = afile.read(BLOCKSIZE)
+        while len(buf) > 0:
+            hasher.update(buf)
+            buf = afile.read(BLOCKSIZE)
+    return hasher.hexdigest()
+
+def package_hash(file_list):
+    sorted(file_list)
+    h = hashlib.sha1("".join(file_list).encode())
+    return h.hexdigest()
+
+def file_writer(o, filepath, filename, sha1, license, copyright='NOASSERTION', comment='NOASSERTION'):
```

Comment:
What is 'o'?

Also - if you're using Python 3 consider using typehints - https://docs.python.org/3/library/typing.html